extension/src/stats_agg.rs

```diff
@@ -209,51 +191,50 @@ pub fn stats2d_trans_inner(
             let val: Option<XYPair> = match (y, x) {
                 (None, _) => None,
                 (_, None) => None,
-                (Some(y), Some(x)) => Some(XYPair{y, x})
+                (Some(y), Some(x)) => Some(XYPair { y, x }),
             };
             match (state, val) {
-                (None, None) => Some(StatsSummary2D::from_internal(InternalStatsSummary2D::new()).into()), // return an empty one from the trans function because otherwise it breaks in the window context
+                (None, None) => {
+                    Some(StatsSummary2D::from_internal(InternalStatsSummary2D::new()).into())
+                } // return an empty one from the trans function because otherwise it breaks in the window context
```

Comment:
nit: could turn this into 
```rust
                    ...
                    // return an empty one from the trans function because otherwise it breaks in the window context
                    Some(StatsSummary2D::from_internal(InternalStatsSummary2D::new()).into())
                 } 
```
if it's not too much trouble

Reply:
Added to reviewer comments commit.